### ISSUE TITLE
UX of "enable predicting" button

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/alchemy#193]
If you have no GEN approved, this button appears in the place of GEN stakes.

I think this is bad UX, because:

* the information about what is been staked is interesting for the user, independently of the fact that I can participate or not
* on the boosted proposals, the button is actually misleading, because ven if you "enable predictions", you will not be allowed to predict on them anyway.

Makes more sense to me to prompt the user to approve GEN for staking at the moment that she actually decides to stake, i.e. just show the same UI independently of the pre-approval, and prompt the user at the moment she clicks "pass" or "fail"

**Comment by tibetsprague** [comment]
Agreed, it was supposed to be very temporary of a UX as we figured out how to not need approval of transfers at all, but that never happened. So the UX needs an upgrade
